javascript/javascript3/week3/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Screenshot Capture</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <div class="main">
+      <form id="create-users">
+        <fieldset>
+          <legend>Create Users</legend>
+          <label for="userEmail">Insert e-mail:</label>
+          <input type="email" name="userEmail" id="userEmail" />
+          <label for="userPassword">Choose password:</label>
+          <input type="password" name="userPassword" id="userPassword" />
```

Comment:
should be required field

Good reading https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation